src/components/Card/carouselcard.css

```diff
@@ -63,6 +63,7 @@ The css colors have to be updated with tailwind utility colors
 
 @screen sm {
   .lil-carousel-card {
+    height: 450px;
```

Comment:
450 ?

Reply:
Was a hacky way to fix card height, removed for now and will take care of it separetly